Impossible to install Kraken

There's a incompatibility avoiding generator-kraken to correctly install Kraken and start server:


```
npm ERR! peerinvalid The package dustjs-linkedin does not satisfy its siblings' peerDependencies requirements!
npm ERR! peerinvalid Peer engine-munger@0.2.7 wants dustjs-linkedin@>= 2.0.0 < 2.7.0
npm ERR! peerinvalid Peer dustjs-helpers@1.7.1 wants dustjs-linkedin@2.7 - 2.8
```

--- a/app/index.js
+++ b/app/index.js
@@ -175,7 +175,9 @@ module.exports = yeoman.generators.Base.extend({
 
             var dependencies = this._dependencyResolver('npm');
             if (dependencies) {
-                this.npmInstall(dependencies, { save: true });
+                forEachDepPrefix.call(this, dependencies, function (deps, type) {
+                    this.npmInstall(deps, { save: true, savePrefix: type });
+                });
             }
         },
 
@@ -187,7 +189,9 @@ module.exports = yeoman.generators.Base.extend({
             var dependencies = this._dependencyResolver('npmDev');
 
             if (dependencies) {
-                this.npmInstall(dependencies, { saveDev: true });
+                forEachDepPrefix.call(this, dependencies, function (deps, type) {
+                    this.npmInstall(deps, { saveDev: true, savePrefix: type });
+                });
             }
         },
 
@@ -260,3 +264,20 @@ module.exports = yeoman.generators.Base.extend({
     }
 
 });
+
+function forEachDepPrefix(dependencies, cb) {
+    var work = {};
+    dependencies.forEach(function (dep) {
+        var char = dep.split('@')[1][0];
+        if (char !== '^' && char !== '~') {
+            char = '^';
+        }
+
+        work[char] = work[char] || [];
+        work[char].push(dep);
+    });
+
+    for (var type in work) {
+        cb.call(this, work[type], type);
+    }
+}

